kitsune/users/auth.py

```diff
@@ -68,9 +68,22 @@ def create_user(self, claims):
         profile.products.clear()
         profile.products.add(*products)
 
-        # This is a new sumo profile, redirect to the edit profile page
-        self.request.session['oidc_login_next'] = reverse('users.edit_my_profile')
-        messages.info(self.request, 'fxa_notification_created')
+        # This is a new sumo profile, show edit profile message
+        messages.success(
```

Comment:
I am not sure if we need to show this on the form page or it would have been better to amend it when the user has posted the question. If the user clicks any of the links in the message then basically we are getting the user out of the flow. Another solution could be to remove completely this part (if the messaging when posting a question for a new user is too much) for new users when in flow.

Reply:
We could _blank the links, to make it clearer the user can come back to the AAQ flow.

Reply:
In that case though you also alter the behavior for new users not in the AAQ flow.